meetings/2021-07/july-15.md

```diff
@@ -0,0 +1,734 @@
+
+JHD: So, yeah, I think the other challenge, related to what we just talked about. That's been talked about in Matrix Is that like, as you said, some of the intrinsics are not reachable off the global. So it's actually not trivial to just Loop over the global and grab everything. So, yeah, I think I agree that that what has been lightly sketched out some sort of function that I can cache and then call that will get you know the previous code could have replaced. But that will get All the originals at that time is, would solve my use case without direct realm object access? 
+
+MM: For sure, I would solve it better. 
+
+JHD: Yeah, I would agree with that, the Yes. If so then what are, what is the suggestion based on that?
+
+MM:  We've identified a need with some concrete ideas of possible API approaches for satisfying the need, I think we've got something that qualifies very well for a new stage one proposal. I think we should address that need there and we should realize that since the need can be addressed and that there's nothing about the realm proposal in its current state that impedes that in any way that we should allow the Realms proposal in its current state to proceed to stage 3.
+
+JHD: so, it cues get accused empty. So I think that seems a reasonable path forward under one - "condition" is the wrong word because that implies leverage but under the one, like in one circumstance, which is that the "direct object access for Realms" path. It does not appear to me that there will ever that will ever be allowed unless the current membership of TC39 completely turns over are so. It's, it would be kind of weird to ask for consensus on stage for stage one on a proposal or a problem that nothing was written up and put on the agenda before. right now. So it'd be nice if that could be granted, But that alone isn't sufficient because the real piece is, I would kind of need to know if any like, in the same way as when realms, Went to stage two. There was, it was, I think clearly telegraphed that there were these potential blocking concerns for the direct object access approach, right? Is there anyone in the room who has any reason to believe that such a proposal modulo, you know, dealing with various sorts of concerns that proposals run across, but that such a proposal would, would not have a path toward stage 4. Does that make sense? What I'm asking, I'm trying to figure out if it's worth me investing my time. 
```

Comment:
```suggestion
JHD: so, the queue is empty. I think that seems a reasonable path forward under one - "condition" is the wrong word because that implies leverage but under one circumstance, which is that the "direct object access for Realms" path does not appear to me that there will ever that will ever be allowed unless the current membership of TC39 completely turns over. It would be kind of weird to ask for consensus on for stage 1 on a proposal or a problem that nothing was written up and put on the agenda before right now. So it'd be nice if that could be granted, but that alone isn't sufficient because the real piece is, I would need to know - in the same way as when realms went to stage 2, it was clearly telegraphed that there were these potential blocking concerns for the direct object access approach, right? Is there anyone in the room who has any reason to believe that such a proposal modulo, you know, dealing with various sorts of concerns that proposals run across, but that such a proposal would, would not have a path toward stage 4. Does that make sense? What I'm asking, I'm trying to figure out if it's worth me investing my time.
```